docs/resources/database.md

```diff
@@ -1 +1,464 @@
-# :resources-database: Database
\ No newline at end of file
+# :resources-database: Database
+
+Database as a [Resource](/resources/) within DataOS is declared for creating a database within DataOS. It is treated as a first-class resource, encapsulating the fundamental unit for storing and organizing structured data using PostgreSQL. The classification of Databases as a resource type within the DataOS ecosystem allows for systematic management, provisioning, and utilization of databases in a standardized manner.
+
+<aside class=”callout”>
+
+ Database is a Workspace-level Resource type.
+
+</aside>
+
+
+<div class="grid cards" markdown>
+
+-   :material-card-bulleted-settings-outline:{ .lg .middle } **Key Features of Database**
```

Comment:
Instead of a grid card for 'Key features' you should create a grid card for 'How to create and manage a Database?'